install/install.sh

```diff
@@ -9,11 +9,47 @@ REMOTE="$REMOTE/$VERSION"
 set -e
 
 # Check if the script is running in ARM architecture
-[[ "$(uname -m)" != "arm"* ]] && echo "Companion only supports ARM computers."
+[[ "$(uname -m)" != "arm"* ]] && (
+    echo "Companion only supports ARM computers."
+    exit 1
+)
 
 # Check if the script is running as root
 [[ $EUID != 0 ]] && echo "Script must run as root."  && exit 1
 
+# Running in a BCM27* or BCM28, do necessary Raspberry configuration
+cat /proc/cpuinfo | grep -Eq "BCM(27|28)" && (
+    echo "Configuring board.."
+
+    # Remove any configuration related to i2c and spi/spi1 and do the necessary changes for navigator
+    echo "- Enable I2C, SPI and UART."
+    for STRING in "dtparam=i2c_arm=" "dtparam=spi=" "dtoverlay=spi1" "dtoverlay=uart1"; do
+        sudo sed -i "/$STRING/d" /boot/config.txt
+    done
+    for STRING in "dtparam=i2c_arm=on" "dtparam=spi=on" "dtoverlay=spi1-3cs" "dtoverlay=uart1"; do
+        echo "$STRING" | sudo tee -a /boot/config.txt
+    done
+
+    # Check for valid modules file to load kernel modules
+    if [ -f "/etc/modules" ]; then
+        MODULES_FILE="/etc/modules"
+    else
+        MODULES_FILE="/etc/modules-load.d/companion.conf"
+        touch "$MODULES_FILE" || true # Create if it does not exist
+    fi
+
+    echo "- Load kernel modules."
```

Comment:
```suggestion
    echo "- Set up kernel modules."
```
They will only be loaded after the reboot, right?